test/methods/polygonize.jl

```diff
@@ -51,19 +55,23 @@ end
         end
         @test GO.equals(data_mp, evil_in_data_space_mp)
     end
+end
+
+@testset "Polygonize with DimensionalData compatible arrays" begin
+    data = rand(1:4, 100, 50) .== 1
+    dd = DD.DimArray(data, (DD.X(51:150), DD.Y(151:200)))
     @testset "DimensionalData" begin
-        data = rand(1:4, 100, 100) .== 1
-        evil = DimensionalData.DimArray(data, (DimensionalData.X(1:100), DimensionalData.Y(1:100)))
-        data_mp = polygonize(data)
-        evil_mp = @test_nowarn polygonize(evil)
-        @test GO.equals(data_mp, evil_mp)
+        data_mp = polygonize(51:150, 151:200, data);
+        dd_mp = polygonize(dd);
+        @test GO.equals(data_mp, dd_mp)
     end
+
     @testset "Rasters" begin
-        data = rand(1:4, 100, 100) .== 1
-        evil = Rasters.Raster(data; dims = (DimensionalData.X(1:100), DimensionalData.Y(1:100)), crs = Rasters.GeoFormatTypes.EPSG(4326))
```

Comment:
Oh no, the evil is gone :D

Reply:
Turned out it wasn't actually evil, just a regular garden variety array